### ISSUE TITLE
Add a SMTP server configuration UI

### ISSUE THREAD

**Issue by simongeorges** [makinacorpus/drupal-netsmtp#1]
To be able to include this module in all "small" websites, or websites for non-coder clients, it would be very nice to add a dedicated UI to configure at least the smtp server.

This issue is just a reminder to do it one day.

**Comment by pounard** [comment]
That's an good idea, I may do it sometime, but there is no clear roadmap, please do it if you need it :)

**Comment by vladdancer** [comment]
No, please, don't do that. There is no problem for non-coders to configure module settings inside settings.php. Besides, it's a good idea to store settings in the settings.php file for easy app deployment (really I hate features module using for deployment),
we use settings.local.php for things for current host and settings.php for global settings.

**Comment by pounard** [comment]
@vladdancer while I sure do agree with you, I sometime like to provide this kind of feature as a convenience for people that don't... The fact is I did started to write this some time ago, in a separate optional module but I never had the chance to finish it, so I'll keep this issue open until I do (it may happen sooner than expected, since I'll go for a round of fixes for a new project).
